Makefile

```diff
@@ -1,30 +1,32 @@
+REBAR = rebar3
```

Comment:
the rebar binary was in the repo. I don't mind which but we can delete the original rebar binary or put the new one in its place.